src/raw.rs

```diff
@@ -134,9 +134,15 @@ bitflags! {
         const EXPIRED = REDISMODULE_NOTIFY_EXPIRED;
         const EVICTED = REDISMODULE_NOTIFY_EVICTED;
         const STREAM = REDISMODULE_NOTIFY_STREAM;
+        #[cfg(any(
+            feature = "min-redis-compatibility-version-7-0",
+            feature = "min-redis-compatibility-version-7-2"
+        ))]
+        const NEW = REDISMODULE_NOTIFY_NEW;
```

Comment:
What if a module which want to be comparable with redis 6 but still want to register to this NEW notification in case it is running on redis 7? Can we think how we can still support this case?

Reply:
I did not mean that the module want to use this flag on Redis 6. I meant that the module want to be Redis 6 compatible and conditionally gets the NEW notification in case it runs on Redis >= 7.0.1.

Reply:
Then we can just remove the `#[cfg]`, I guess. Redis should check the flags too, if the module doesn't do that.